src/utils/appDataClient.ts

```diff
@@ -58,13 +58,13 @@ const breadcrumbOpts = { type: 'HTTP Request', category: 'appDataClient' }
 
 export const configChainVars = async () => {
   Logger.breadcrumb('configChainVars', breadcrumbOpts)
-  const vars = await client.vars.get()
+  const vars = await client.vars.getTransactionVars()
   Transaction.config(vars)
 }
 
-export const getChainVars = async () => {
+export const getChainVars = async (keys?: string[]) => {
```

Comment:
It would be nice to have some typing here or in helium-js
```ts
type ChainVar =
  | 'num_consensus_members'
  | 'transfer_hotspot_stale_poc_blocks'
  | 'staking_fee_txn_assert_location_dataonly_gateway_v1'
export const getChainVars = async (keys?: ChainVar[]) => {
```

Reply:
Next time I’m in helium-js I’ll add it. There are a lot of keys and they may change though